stats/stats_test.go

```diff
@@ -81,6 +81,50 @@ var (
 	}
 	// The id for which the service handler should return error.
 	errorID int32 = 32202
+	// To verify if the Unary RPC server stats events are logged in the
+	// correct order.
+	expectedUnarySequence = []string{
```

Comment:
this can be initialized within the test itself because we are not using it anywhere else. Also, convention is to use `want/wanted` instead of `expected`

Reply:
Got it! I will keep this in mind going forward. Thank you!